privacy policy updated: blogpost + new privacy policy in english/german (#482)
deltachat/deltachat-pages: _posts/2021-10-06-blog-GDPR.md

---
title: New data protection officer appointed and refined privacy policies
author: jette, treefit, missytake and holga
image: ../assets/blog/fabian-schmieder.jpeg
---

> It has been a while since the last update on Delta Chat privacy policy and some things have changed. We appointed Prof.Dr. Fabian Schmieder of [University of Applied Sciences and Arts Hanover](https://im.f3.hs-hannover.de/en_us/studium/personen/prof-dr-fabian-schmieder/) who is now our data protection officer. He has been helping us for a long time and is not only a great advisor but also a brilliant pianist!:)

Reviewer: ```suggestion
A while has passed since the last Delta Chat privacy-policy update, and some things have changed. Prof.Dr. Fabian Schmieder of [University of Applied Sciences and Arts Hanover](https://im.f3.hs-hannover.de/en_us/studium/personen/prof-dr-fabian-schmieder/) has been appointed our data protection officer (DPO). He has been a helping a helping hand for a long time and is not only a great advisor, but also a brilliant pianist!:)
```

> It has been a while since the last update on Delta Chat privacy policy and some things have changed. We appointed Prof.Dr. Fabian Schmieder, professor for data protection and media law of the [University of Applied Sciences and Arts Hanover](https://im.f3.hs-hannover.de/en_us/studium/personen/prof-dr-fabian-schmieder/) and coauthor of the standard work [Betrieblicher Datenschutz (Operational data protection)](https://www.beck-shop.de/forgo-helfrich-schneider-betrieblicher-datenschutz/product/24093138), who is now our data protection officer. He has been helping us for a long time and is not only a great advisor but also a brilliant pianist!:)

Reviewer 2: ```suggestion
It has been a while since the last update on Delta Chat privacy policy and some things have changed. We appointed Prof. Dr. Fabian Schmieder, professor for data protection and media law of the [University of Applied Sciences and Arts Hanover](https://im.f3.hs-hannover.de/en_us/studium/personen/prof-dr-fabian-schmieder/) and coauthor of the standard work [Betrieblicher Datenschutz (Operational data protection)](https://www.beck-shop.de/forgo-helfrich-schneider-betrieblicher-datenschutz/product/24093138), who is now our data protection officer. He has been helping us for a long time and is not only a great advisor but also a brilliant pianist!:)
```

> Jointly we worked with Kai Korte from [LexICT](https://lexict.de) to professionalize and refine our privacy policy. We split the privacy policy into one for the [apps](gdpr), and another separate one for the optional use of the [website](gdpr-website) and community services.

Reviewer: ```suggestion
In conjunction with Kai Korte of [LexICT](https://lexict.de) the privacy policy has been further refined. It is split between the [apps](gdpr), and another for the optional use of the [website](gdpr-website) and community services.
```

> As the Russian surveillance authority Roskomnadzor [learned last year](https://twitter.com/delta_chat/status/1256137319150751744) our apps are e-mail apps and do not contact servers other than user-chosen e-mail providers. But if you choose to interact with our website, [support forum](https://support.delta.chat), payment or social media presences, you can read up on our web privacy policy for details. TLDR; no cookies, no tracking, no ads. You are welcome :)

Reviewer: ```suggestion
As an authority on oversight, the Russian surveillance authority Roskomnadzor least year learnt [our app is an e-mail client last year year](https://twitter.com/delta_chat/status/1256137319150751744), meaning it does not contact servers other than providers of e-mail services chosen by the user. In similar fashion, no cookies, tracking or ads are employed when interacting our website, [support forum](https://support.delta.chat), payment, or social media presences.
```

Author: I do not quite understand your suggestion concerning Roskomnadzor and think that the other suggestions could interfere with DCs general posting style. Thanks anyway!

> If you have any questions or comments please file issues on github 

Reviewer: ```suggestion
Questions and comments can be forwarded as issues on [GitHub](https://github.com/deltachat/deltachat-pages/issues)
```

> or send a message to delta-privacy@merlinux.eu 

Reviewer: ```suggestion
or sending an Autocrypted message to delta-privacy@merlinux.eu 
```

---
title: We updated our GDPR for our apps and website!
author: jette, treefit and missytake
image: ../assets/blog/privacy-policy-certificate.png
---

It has been a while since the last update on Delta Chat privacy policy and some things have changed. We are still trying to keep personal data processing to a minimum and stick to our principle "privacy by design", but as you know, the internet does not work without data. 

> 

Reviewer 3: > The idea for now is to wait until @r10s releases the new iOS features

not sure i understand the idea - the heartbeat-feature is long released (since 2021-03, v1.17 :) 

there was sidenote at https://delta.chat/en/2021-02-04-ios-is-catching-up#what-else, all in all, notifications are more reliable than before, ppl report in that direction, however, apple may still throttle notifications, so we can still not give guarantees.

technically, we send a "deviceToken" provided by Apple to our server. our server pings Apple with that deviceToken every 20 minutes then. Apple may or may not wake up the Delta Chat app then. We do not attach anything to the deviceToken.

> 

Reviewer: Does the iOS heartbeet "feature" run afoul of the "in similar fashion" statement below?

> 

Author: > > The idea for now is to wait until @r10s releases the new iOS features
> 
> not sure i understand the idea - the heartbeat-feature is long released (since 2021-03, v1.17 :)
> 
> there was sidenote at https://delta.chat/en/2021-02-04-ios-is-catching-up#what-else, all in all, notifications are more reliable than before, ppl report in that direction, however, apple may still throttle notifications, so we can still not give guarantees.
> 
> technically, we send a "deviceToken" provided by Apple to our server. our server pings Apple with that deviceToken every 20 minutes then. Apple may or may not wake up the Delta Chat app then. We do not attach anything to the deviceToken.

@hpk42 said that you'll soon release new apple features and we just thought that the heartbeat feature was not mentioned by itself in a blog, so we thought about mentioning it again once you release the new apple features and write a blog post about it anyway.

> 

Author: closes #482 
The final step to do is to manually copy the updated privacy policies in English to all the other languages again since there were changes in between. And check if the German version is the up to date one. 
@missytake or @Simon-Laux: maybe one of you could do this because then we are all set for publishing our policies and the blog! That would be great! Thanks a lot :)
